data/basal.go

```diff
@@ -0,0 +1,166 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
+	"github.com/tidepool-org/platform/validate"
+)
+
+func init() {
+	validator.RegisterValidation(rateTag, BasalRateValidator)
+	validator.RegisterValidation(durationTag, BasalDurationValidator)
+	validator.RegisterValidation(deliveryTypeTag, BasalDeliveryTypeValidator)
+	validator.RegisterValidation(insulinTag, BasalInsulinValidator)
+	validator.RegisterValidation(valueTag, BasalValueValidator)
+}
```

Comment:
Will some of these validators be promoted to more general usage if they are needed by other types?